ledger/internal/eval.go

```diff
@@ -1267,8 +1267,6 @@ func (eval *BlockEvaluator) endOfBlock() error {
 		return err
 	}
 
-	eval.state.mods.OptimizeAllocatedMemory(eval.proto)
```

Comment:
why are we removing this?

Reply:
because we normally have only 8 deltas in mem and not worth optimizing storage for them. This function used to be a temp workaround.